src/tilegarden/src/api.js

```diff
@@ -77,6 +77,10 @@ const writeToS3 = (tile, req) => {
             const { z, x, y, job_id, config } = req.pathParameters
             key = `tile/${job_id}/${config}/${z}/${x}/${y}`
             /* eslint-enable camelcase */
+        } else if (key.startsWith('/')) {
+            // API Gateway request.path object starts with a leading slash,
+            // which would cause the uploaded object to have the wrong prefix.
+            key = key.slice(1)
```

Comment:
While debugging, I noticed that cache tiles were getting written to the S3 bucket in an empty root directory (showing up as a whitespace character for the directory name). I'm pretty sure this is due to another difference between the API Gateway request object representations in `claudia-local-api` and AWS.

Reply:
Good catch and thanks for fixing it up.